src/shmem_atomic.h

```diff
@@ -29,6 +29,19 @@
 #define likely(x)       __builtin_expect((x),1)
 #define unlikely(x)     __builtin_expect((x),0)
 
+#define SHMEM_INTERNAL_INCLUDE
+#include "shmem.h"
```

Comment:
Why was this added?  We should only need to include ```shmem.h``` in the .c files that define the C language bindings.

Reply:
If we do not include ```shmem.h```, then the build reports this error. 
```
./shmem_atomic.h:40:69: error: 'SHMEM_THREAD_SINGLE' undeclared (first use in this function)
 # define SHMEM_INTERNAL_NEED_MEMBAR (shmem_internal_thread_level != SHMEM_THREAD_SINGLE)
```
Please let me know how else I can resolve this.

Reply:
Oh, right!  This #include is ok to keep.